apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/Rates/Rate.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { useCallback } from "react";
 import styled from "styled-components";
-import { Text } from "@ledgerhq/react-ui";
+import { Text, ProviderIcon } from "@ledgerhq/react-ui";
 import Box from "~/renderer/components/Box";
 import { ExchangeRate } from "@ledgerhq/live-common/exchange/swap/types";
-import { iconByProviderName } from "../../utils";
+
```

Comment:
Sorry, but perhaps the jira was not clear enough. We need to remove any ProviderIcon  dependency from @ledgerhq/react-ui

Reply:
Similar to comment above. I've reworked this, however it does still load asset from CDN just not from `@ledgerhq/react-ui`.